package-lock.json

```diff
@@ -89,7 +89,7 @@
     },
     "gateway-js": {
       "name": "@apollo/gateway",
-      "version": "0.38.0",
```

Comment:
Expected based on our publishing pipeline previously not handling top level `package-lock` updates during release.